www/src/pages/tutorial.js

```diff
@@ -7,44 +7,19 @@ import Container from "../components/container"
 export default () => (
   <Container>
     <Helmet>
-      <title>Tutorial</title>
+      <title>Getting Started</title>
     </Helmet>
-    <h1 css={{ marginTop: 0 }}>Gatsby.js Tutorial</h1>
+    <h1 css={{ marginTop: 0 }}>Getting Started</h1>
     <p>
-      Hi! We’re so happy you decided to try using Gatsby. This tutorial has four
-      parts that'll walk you from getting started developing and building Gatsby
-      sites to deploying a finished and polished high performance static PWA.
-    </p>
-    <p>
-      This tutorial is for <em>everyone</em>! You do not need to be a programmer
-      or React.js expert. We'll walk you through things.
-    </p>
+      Hi! 
+      
+      Depending how new you are to Gatsby, you can try one of the following:
```

Comment:
I'd remove the "Depending how new your are to Gatsby" part — there's lots of reasons people go one way or another that aren't due to how new they are to Gatsby. Just present the options.

Also, are you planning on changing the "get started" page? https://www.gatsbyjs.org/docs/

Actually... looking at the home page again — I'm not sure we need to change much for the getting started flow. We have a link on the front page to "Get Started" which tells how to install the cli and immediately build something. The top nav has links to the docs and tutorial. The "Get Started" page also directs people to the tutorial.

Reply:
So, the flow is a bit funky from the home page. Several people I've interviewed and other UX people have mentioned--why are tutorials and Get Started in two different places? I'm not sure exactly what to do about that. My guess is that the "Get Started" link should funnel people towards this portal with 3 choices--tutorials, quick start, docs, although having tabs across the top is really nice too. I haven't seen many sites that do a better job. I figure we can leave that decision for another day.

Reply:
Hmm interesting — my impression is most larger projects do split the getting started and the tutorial. A quick "get started" page is targeted towards advanced users or people already familiar with the project but who want a quick refresher on how to do the initial setup. E.g. you've used Gatsby at home but are setting things up at work and forgot what the CLI tool is called.

Reply:
We should perhaps mention at the top of the "get started" page the tutorial and docs to direct people that way.

Reply:
I think the difference in our case is that many of the "get started" are more of a "quickstart" (e.g. 5 minutes to install and get something running). Gatsby's get started is way more in-depth, if I'm not crossing my wires here.

Reply:
It should only take you five minutes to go through the instructions here: https://www.gatsbyjs.org/docs/ (unless you don't have node or something installed)

Reply:
People I've observed--when they get to the homepage of Gatsbyjs.org, invariable click on "Get Started" first. Then they're like, whoa. I don't understand this. OR, they install stuff, and then they're like, "hm, I need more guidance." Then they look around, and find the tutorial.